### ISSUE TITLE
Not compatible ie11

### ISSUE THREAD

**Issue by beidan** [vercel/swr#278]
some code  didnot transform to es5 ....

**Comment by eivindml-solvr** [comment]
We are having the same issue. The error reads:

```
Object doesn't support property or method 'repeat'
```

The app works in IE11, but when adding `swr` it crashes. Tried various polyfills without any luck. 

<img width="1387" alt="Skjermbilde 2020-02-24 kl  10 20 16" src="https://user-images.githubusercontent.com/55379708/75140459-41d6f380-56ef-11ea-9dd9-e39b8808b973.png">

**Comment by embeddedt** [comment]
[String.prototype.repeat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat) is part of ES6/ES2015; you would have to add an appropriate polyfill for browsers which don't support ES6.

**Comment by eivindml-solvr** [comment]
I have tried to add that polypill multiple times, without any luck. 

It's added at the top of the `index.js` and copied directly from the Mozilla code.
<img width="1253" alt="Skjermbilde 2020-02-25 kl  09 36 14" src="https://user-images.githubusercontent.com/55379708/75229574-7d86c180-57b2-11ea-8d80-dac09cc5cbf0.png">

IE11 still gives the same error, and I have verified that the polyfill is being executed.

<img width="1443" alt="Skjermbilde 2020-02-25 kl  09 36 54" src="https://user-images.githubusercontent.com/55379708/75229639-a1e29e00-57b2-11ea-96d7-49307357e297.png">

**Comment by beidan** [comment]
`{
                test: /\.(js|tsx|ts)?$/,
                include: [path.join(process.cwd(), 'node_modules/swr')],
                loaders: [{
                    loader: 'babel-loader'
                }]
 },`

i use bable to transform it. and it work in ie11.

**Comment by joris-** [comment]
Hi ! 
Did it works for you ?
I don't have same error. I'm using swr with Next.js.
Thanks for help

![image (2)](https://user-images.githubusercontent.com/7545547/79695594-0d5b6000-8278-11ea-9a24-38df38b3452e.png)

**Comment by quangdng** [comment]
@joris- 
Have a look at https://www.npmjs.com/package/next-transpile-modules if you are using Next.js. This helps transpile untranspiled modules in `node_modules`. 

In your next.config.js
`
const withTM = require("next-transpile-modules")([
  "swr"
]);

module.exports = withTM();
`

**Comment by joris-** [comment]
> @joris- 
> Have a look at https://www.npmjs.com/package/next-transpile-modules if you are using Next.js. This helps transpile untranspiled modules in `node_modules`. 
> 
> In your next.config.js
> ```
> const withTM = require("next-transpile-modules")([
>   "swr"
> ]);
> 
> module.exports = withTM();
> ```

Thx I'll try

**Comment by morriq** [comment]
@beidan @joris- 

This one works for me:

```
      {
        test: /\.(js|jsx)?$/,
        exclude: /node_modules\/(?!swr)/,
        use: {
          loader: 'babel-loader',
        },
      },
```
based on: https://github.com/babel/babel-loader/issues/171

**Comment by shuding** [comment]
A fix is released in v0.2.3 :tada:
